openmetadata-spec/src/main/resources/json/schema/type/customProperties/enumConfig.json

```diff
@@ -3,7 +3,7 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "EnumConfig",
   "type": "object",
-  "javaType": "org.openmetadata.schema.type.customproperties.EnumConfig",
+  "javaType": "org.openmetadata.schema.type.customProperties.EnumConfig",
```

Comment:
why are we changing this class path?

Reply:
because the java package generated is customProperties (and not `customproperties`)